recipes/pybind11_json/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, tools
+import os
+
+
+class Pybind11JsonConan(ConanFile):
+    name = "pybind11_json"
+    homepage = "https://github.com/pybind/pybind11_json"
+    description = "An nlohmann_json to pybind11 bridge"
+    topics = (
+        "conan",
+        "header-only",
+        "json",
+        "nlohmann_json",
+        "pybind11",
+        "pybind11_json",
+        "python",
+        "python-binding",
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.9.1")
+        self.requires("pybind11/2.6.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "pybind11_json-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        self.copy(
+            "*", dst="include", src=os.path.join(self._source_subfolder, "include")
+        )
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "pybind11_json"
```

Comment:
I can't find any reference to a pc file in their cmake script: https://github.com/pybind/pybind11_json/blob/master/CMakeLists.txt
```suggestion
```

Reply:
@cbachhuber 
This is not removed yet.

Reply:
This line is removed, I applied your suggestion.

Did you want to remove line 

```python
        self.cpp_info.names["pkg_config"] = "pybind11_json"
```

instead? That fits better to your comment mentioning a pc file.